trio/_dtls.py

```diff
@@ -841,10 +841,7 @@ def __init__(self, endpoint: DTLSEndpoint, peer_address: Any, ctx: Context):
         # support and isn't useful anyway -- especially for DTLS where it's equivalent
         # to just performing a new handshake.
         ctx.set_options(
-            (
-                SSL.OP_NO_QUERY_MTU
-                | SSL.OP_NO_RENEGOTIATION  # type: ignore[attr-defined]
-            )
+            SSL.OP_NO_QUERY_MTU | SSL.OP_NO_RENEGOTIATION  # type: ignore[attr-defined]
```

Comment:
Huh, why did ruff change this? I think it was intentionally formatted to showcase which specific attribute wasn't defined.

Reply:
I saw this and the diff and was a bit confused myself, not sure about this one.

Reply:
Another one of my PRs hit this and I looked into it further, this change was by black, not ruff.